recruiter-email.txt

```diff
@@ -0,0 +1,14 @@
+<Name>,
+
+Just wanted to reach out about your first CoffeeOps meet-up last Friday.
+
+Seattle CoffeeOps is a community-driven meet-up run by engineers. As such, we try to keep our meetings non-commercial and pitch-free. I'm sorry I didn't warn you of that when you gave me your staffing pitch at the beginning of the meeting.
+
+We do welcome all folks (beginners and experienced folks) who want to actively contribute to building the community. We also have a couple of recruiters who attend regularly, listen, and participate in discussions.
```

Comment:
Suggestion:

We also have _**some recruiters and others in non-technical roles**_ who attend regularly, listen, and participate in discussions. _**There are many opportunities for networking, and forming meaningful relationships that may result in referrals/sales/follow-up outside of CoffeeOps, but we are a community first and foremost.**_